flink-connectors/flink-connector-hive/src/main/java/org/apache/flink/connectors/hive/HiveTableSink.java

```diff
@@ -170,7 +171,7 @@ public void setStaticPartition(Map<String, String> partitionSpec) {
 		staticPartitionSpec = new LinkedHashMap<>();
 		for (String partitionCol : getPartitionFieldNames()) {
 			if (partitionSpec.containsKey(partitionCol)) {
-				staticPartitionSpec.put(partitionCol, partitionSpec.get(partitionCol));
+				staticPartitionSpec.put(partitionCol, PlanUtils.stripQuotes(partitionSpec.get(partitionCol)));
```

Comment:
Just curious, partitionSpec is a map of strings, and the value should probably have been unquoted before getting here. I'm wondering if the fix should be at a lower level than here. Otherwise, similar fix will be needed wherever such partition spec is used.